planner-api/openapi.yaml

```diff
@@ -81,6 +81,48 @@ paths:
         "401":
           description: "User unauthorized"
 
+  "/api/goal":
+    post:
+      summary: "Create goal"
+      requestBody:
+        description: "Goal data"
+        required: true
+        content:
+          "application/json":
+            schema:
+              "$ref": "#/components/schemas/GoalCreateRequest"
+      responses:
+        "200":
+          description: "Goal successfully created"
+        "401":
+          description: "User unauthorized"
+
+  "/api/goals":
```

Comment:
Using api would be easier if the path to related operations was the same. Eg. /api/goal. Anyways, we should decide on path convention in requests, so you can ignore this for now.